src/send.c

```diff
@@ -376,6 +381,7 @@ int send_run(sock_t st, shard_t *s)
 			zconf.probe_module->make_packet(
 			    buf, &length, src_ip, current_ip,
 			    htons(current_port), ttl, validation, i,
+			    (uint16_t)aesrand_getword(aes_rand_gen),
```

Comment:
This does an additional extra full blown AES block encryption for every single packet, in order to produce 16 bits of IP ID.  In my quick perf testing on a 10GbE box with 8 cores and netmap, this change reduces send rate from 10.44 Mp/s to 8.80 Mp/s.

Would a more lightweight PRNG make more sense here, think cyclic group like for target permutation, instead of the AES-based PRNG?

Reply:
Hmm, that sounds like a significant perf. hit, thanks for running those numbers! Let me look at grabbing the last 16 bits from permutation like @zakird mentioned, I'll double-check no probe-module uses those.

Reply:
fwiw, 46fac21 resolves the perf hit for me.